asciidoc/action.yml

```diff
@@ -0,0 +1,102 @@
+name: Build Presentations with AsciiDoc
+description: "Generate a Slidedeck from AsciiDoc Markdown"    
+inputs:
+  srcdir:
+    description: "AsciiDoc sourcefile directory"
+    required: false
+    default: "srcfiles"
+  assets:
+    description: "Site assets directory"
+    required: false
+    default: "static"
+  dest_branch:
+    description: "Slidedeck Output branch"
+    required: false
+    default: "presentations"
+  publish:
+    description: "Publish the site to the destination branch. If false, the site is built but not published."
+    required: false
+    default: "true"
+  output:
+    description: "AsciiDoc output directory"
+    required: false
+    default: 'output'
+  tempdir:
+    description: "Temporary Directory name"
+    required: false
+    default: 'output.tmp'
+  debug:
+    description: "AsciiDoc Debug mode"
+    required: false
+    default: 'false'
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v4
```

Comment:
```suggestion
    - uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4.1.0
```
While it's an offical action and this doesn't require pinning in workflows I think it still makes sense within an action as the user has no control over it?

Reply:
would there be a way to test this / keep this hash up to date with dependabot or something? Trying to prevent having to touch this everytime there's an update.

Reply:
yes dependabot can do it, just add the dir with action.yml to the `directories` list in dependabot.yml